src/models/User.ts

```diff
@@ -0,0 +1,21 @@
+import { Schema, model, connect, Model } from 'mongoose';
```

Comment:
```suggestion
import { Schema, model } from 'mongoose';
```

We don't use the `connect` or `Model` imports, so no point including them.